amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/declaration/external/raml/RamlJsonSchemaExpression.scala

```diff
@@ -291,7 +303,7 @@ case class RamlJsonSchemaExpression(key: YNode,
     }
   }
 
-  private def normalizePath(rawPath: String): String = {
+  private def stripPointsAndFragment(rawPath: String): String = {
```

Comment:
Why are we doing this?

Reply:
Appears to be coming for here: https://www.mulesoft.org/jira/browse/APIMF-1840. The raw path cannot be simply resolved against the current file location because is can be an absolute reference (only applies for raml).